pkg/deploy/api/v1beta3/conversion.go

```diff
@@ -33,35 +28,98 @@ func convert_api_DeploymentConfig_To_v1beta3_DeploymentConfig(in *newer.Deployme
 	if err := s.DefaultConvert(in, out, conversion.IgnoreMissingFields); err != nil {
 		return err
 	}
-	if err := s.Convert(&in.Template, &out.Spec, 0); err != nil {
+	if err := convert_api_DeploymentConfigSpec_To_v1beta3_DeploymentConfigSpec(&in.Spec, &out.Spec, s); err != nil {
 		return err
 	}
-	if err := s.Convert(&in.Triggers, &out.Spec.Triggers, 0); err != nil {
+	if err := convert_api_DeploymentConfigStatus_To_v1beta3_DeploymentConfigStatus(&in.Status, &out.Status, s); err != nil {
 		return err
 	}
-	out.Status.LatestVersion = in.LatestVersion
-	if err := s.Convert(&in.Details, &out.Status.Details, 0); err != nil {
+	return nil
+}
+
+func convert_v1beta3_DeploymentConfigSpec_To_api_DeploymentConfigSpec(in *DeploymentConfigSpec, out *newer.DeploymentConfigSpec, s conversion.Scope) error {
```

Comment:
Hm, actually all this code is here because we were doing manual conversion from the internal to out and vice versa. After running the generator though, all the conversions have been updated with these conversions. So should I drop manual? @ironcladlou ?